src/Chainweb/Mempool/InMem.hs

```diff
@@ -244,20 +249,30 @@ insertCheckInMem cfg lock txs
 validateOne
     :: forall t a
     .  InMemConfig t
+    -> ChainwebVersion
     -> HashMap TransactionHash a
     -> Time Micros
     -> t
     -> TransactionHash
     -> Either InsertError t
-validateOne cfg badmap (Time (TimeSpan now)) t h =
-    sizeOK
+validateOne cfg v badmap nw@(Time (TimeSpan now)) t h =
+    transactionsEnabled
+    >> sizeOK
     >> ttlCheck
     >> notInBadMap
     >> _inmemPreInsertPureChecks cfg t
   where
     txcfg :: TransactionConfig t
     txcfg = _inmemTxCfg cfg
 
+    -- | KILLSWITCH: This is to be removed in a future version of Chainweb. This
+    -- prevents any transaction from entering the mempool.
+    --
+    transactionsEnabled :: Either InsertError ()
+    transactionsEnabled = case txSilenceDates v of
+        Just (start, _) | start < nw -> Left InsertErrorTransactionsDisabled
+        _ -> pure ()
+
```

Comment:
I think this isn't enough. Someone could write a hacked node that mines blocks with tx. We should reject those, too. 

Since `txSilentDates` is already a property of the `ChainwebVersion` we can add a payload validation rule ala

```Haskell
prop_silentDates hdr payload = case txSilenceDates v of
    Just (start, _) | start < blockCreationTime hdr -> length (playloadTransactions payload) == 0
    _ -> True
```